src/gridcoin/researcher.cpp

```diff
@@ -664,7 +664,7 @@ AdvertiseBeaconResult GenerateBeaconKey(const Cpid& cpid)
     // help identify transactions in the GUI:
     //
     const std::string address_label = strprintf(
-        "Beacon Rain Address for CPID %s (at %" PRIu64 ")",
+        "Beacon Address for CPID %s (at %" PRIu64 ")",
```

Comment:
If we don't want to keep the word "rain", we should delete the comment above this line. I added "rain" because people were always confused by rain transactions that showed up as the beacon addresses. I saw several instances where people thought they received research rewards. The "rain" label helps to distinguish those transactions from stakes and makes it easier for helpers to provide support for those questions since there is nothing that explicitly categorizes a transaction as rain. I've seen less confusion about the label now than before Fern, but maybe that's just me.

Reply:
I don't think we can win here. We have also received reports of people confused by the presence of Rain in the address field. It is more accurately the beacon address anyway, not just the address that rains are sent to.

Reply:
Yeah, it's a tough one. I think "rain" is a net positive, but I trust your judgement.

Reply:
I changed the comment.